### PR TITLE
align to spec version 0cbc825

### PR DESCRIPTION
**Pull request by nkryuchkov**
port changes between https://github.com/ssvlabs/ssv-spec/commit/7adef2ac3ea6529bed1c52a70469e365733e5ec2 and https://github.com/ssvlabs/ssv-spec/commit/0cbc8252c0fe9fc0b60528e6edea8d51cdea8bd3:

https://github.com/ssvlabs/ssv-spec/compare/tmp-160724-spec-align...alan?expand=1

### CODE REVIEW

File: exporter/api/msg.go [hunk 1]
```diff
@@ -26,7 +27,7 @@ type SignedMessageAPI struct {
 	Signers   []spectypes.OperatorID
 	Message   specqbft.Message
 
-	FullData *spectypes.ConsensusData
+	FullData *spectypes.ValidatorConsensusData
```
[the remark this answers is not in the document]
Comment (moshe-blox) [reply exporter/api/msg.go:30]: @guym-blox wdyt?

[the remark this answers is not in the document]
Comment (y0sher) [reply exporter/api/msg.go:30]: @moshe-blox @MatheusFranco99 
`SignedMessageAPI` is never used anymore, it belongs to old exporter

[the remark this answers is not in the document]
Comment (nkryuchkov) [reply exporter/api/msg.go:30]: but we use `FullData` in ParticipantsAPI

[the remark this answers is not in the document]
Comment (nkryuchkov) [reply exporter/api/msg.go:30]: > but we use `FullData` in ParticipantsAPI

However, we use it like this:

```go
FullData: &spectypes.ValidatorConsensusData{Duty: spectypes.ValidatorDuty{Slot: msg.Slot}},
```

So we just fill the slot and don't use other fields. So I think we can leave it as it

[the remark this answers is not in the document]
Comment (nkryuchkov) [reply exporter/api/msg.go:30]: @MatheusFranco99 please resolve the conversation if you agree or add a comment if you disagree

[the remark this answers is not in the document]
Comment (nkryuchkov) [reply exporter/api/msg.go]: BTW I deleted `SignedMessageAPI` as unused

**Review by y0sher — APPROVED** [review]

**Pull request merged**